bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/app/Dataman.scala

```diff
@@ -66,14 +71,24 @@ object Dataman {
 
     val pollSummit   = GsaPollActions(config.summitHost, config.site, odb)
     val pollArchive  = GsaPollActions(config.archiveHost, config.site, odb)
+    val allPolls     = List(pollSummit, pollArchive)
+
+    val pollService  = PollService(25) { id =>
```

Comment:
I shouldn't have removed the note that accompanied the previous version.  I don't know how many threads are appropriate for a large IO-bound collection of tasks so I arbitrarily chose 25.  To make it clearer what is going on, I'll switch it to:

````scala
PollService(workerCount = 25) { id =>
````